.buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py

```diff
@@ -75,6 +75,22 @@ def dbt_example_extra_cmds_fn(_):
     ]
 
 
+def docs_snippets_extra_cmds_fn(_):
+    return [
+        "pushd examples/docs_snippets",
+        # Run the postgres db. We are in docker running docker
+        # so this will be a sibling container.
+        "docker-compose up -d --remove-orphans",  # clean up in hooks/pre-exit
```

Comment:
there is supposed to be a corresponding addition in `.buildkite/hooks/pre-exit` - i guess this comment needs to be clearer and louder

Reply:
that said, what docs snippets need postgres? seems odd that we need this

Reply:
in a mode/resource tutorial, we mentioned postgres so we have tests covering that